conform-pr/main.go

```diff
@@ -87,9 +93,33 @@ type checkResult struct {
 	err   error
 }
 
-// runChecks runs all the checks on the given PR GraphQL node.
-func runChecks(ctx context.Context, action *githubactions.Action, client *graphql.Client, nodeID string) []checkResult {
-	pr := client.GetPullRequest(ctx, nodeID)
+// runChecks runs all the checks for the given PR.
+//
+//nolint:lll // package and type names are long
+func runChecks(ctx context.Context, action *githubactions.Action, client *github.Client, gClient *graphql.Client, org, user, nodeID string) ([]checkResult, bool) {
```

Comment:
Could we split that to multiple lines? WDYT?

Reply:
Fixed in a different way